### PR TITLE
Corrected the path since the .py files of services are stored in ./pi…

### PR DESCRIPTION
**Pull request by bangpradyumna**
…pot/services

This now makes it possible to delete any service (was giving an error before)

### CODE REVIEW

**Comment by canihavesomecoffee** [comment]
@pep8speaks suggest diff

**Comment by bangpradyumna** [comment]
@canihavesomecoffee No response from the bot yet, does it mean that no changes are required?

**Comment by canihavesomecoffee** [comment]
With the minimal change in this PR, there seem to be no issues indeed. I'll review the actual contents in a minute.

File: mod_config/models.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import json
 import os
-
```
Comment (canihavesomecoffee) [inline mod_config/models.py]: Please don't remove this line? :+1:

**Review by canihavesomecoffee — CHANGES_REQUESTED** [review]

File: mod_config/models.py (hunk 1 above)
Comment (bangpradyumna) [reply mod_config/models.py]: @canihavesomecoffee Added a newline. I think the request can now be merged.

**Review by canihavesomecoffee — APPROVED** [review]

**Pull request merged**